### PR TITLE
NW4 -Gulnihal Naldoken - HTML/CSS - Week 1

### PR DESCRIPTION
**Pull request by gulnihaln**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

-----
[View rendered .github/pull_request_template.md](https://github.com/gulnihaln/HTML-CSS-Coursework-Week1/blob/master/.github/pull_request_template.md)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -6,3 +6,131 @@
...
+article {
+    display: grid;
+    align-items: center;
+    justify-items: left;
+    
+
+}
+
+article:first-child {
+    background-color:rgba(253, 253, 253, 0.5);
```
Comment (zeinaji) [inline css/style.css:96]: I think at the moment this is not working, the use of it is correct but you just need a space between `article` and the `:` so `article :first-child` :)

File: css/style.css [hunk 2]
```diff
@@ -6,3 +6,131 @@
...
  */
+
+ /*full page background color */
+ html {
+    background-color:rgb(221, 240, 240)
+ }
+
+ /*header styling*/
+ .header-container {
+
```
Comment (zeinaji) [inline css/style.css:17]: Just make sure you get rid of the extra white space here

File: index.html [hunk 3]
```diff
@@ -15,8 +15,72 @@
...
   </head>
-  <body>
+
+  
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-  </body>
+      
+      <body>
+      <div class="header-container">
```
Comment (zeinaji) [inline index.html:24]: Just be careful about the indentation here, `<body>` should be further to the left of `<div>`

**Review by zeinaji — COMMENTED** [review]
This is great work Gulnihal, well done 🙂 HTML semantics are used properly and throughout and great job with styling and using flexbox! I know CSS is always a pain 😅

Just make sure you get rid of extra whitespace and use indentation to make things easier to read.

File: css/style.css (hunk 2 above)
Comment (gulnihaln) [reply css/style.css:17]: Do you mean the empty space, line 17?

File: css/style.css (hunk 1 above)
Comment (gulnihaln) [reply css/style.css:96]: Thank you Zein for your comments and showing me how to make it better !

**Comment by gulnihaln** [comment]
> This is great work Gulnihal, well done 🙂 HTML semantics are used properly and throughout and great job with styling and using flexbox! I know CSS is always a pain 😅
> 
> Just make sure you get rid of extra whitespace and use indentation to make things easier to read.

Thank you @zeinaji for your comments and for showing me how to make it better!

File: css/style.css (hunk 2 above)
Comment (zeinaji) [reply css/style.css:17]: Yeah 🙂

**Pull request closed without merging**